src/main/java/core/basesyntax/service/ShopService.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ShopService {
+    void process(List<FruitTransaction> transactions);
+
+    Storage getStorage();
```

Comment:
Following the encapsulation principle, it's not a good practice to expose the internal details of a class, such as the `Storage`. Instead, the `ShopService` should provide operations to manipulate the storage without directly exposing it.